### PR TITLE
nightwave&baro updates

### PR DESCRIPTION
**Pull request by cagatayldzz**

### CODE REVIEW

File: data/languages.json [hunk 1]
```diff
@@ -8896,7 +8896,7 @@
     "value": "Prisma Avia Chest Plate"
   },
   "/lotus/storeitems/upgrades/skins/armor/furisarmor/prismafurislarmor": {
-    "value": "Prisma Avia Leg Plates"
+    "value": "Prisma Avia Leg Plate"
```
Comment (TobiTenno) [inline data/languages.json]: Should be `Ankle`, not `Leg`

File: data/languages.json [hunk 2]
```diff
@@ -5308,7 +5308,7 @@
     "value": "Charger Specter"
   },
   "/lotus/storeitems/types/restoratives/consumable/corruptedheavygunnerball": {
-    "value": "Corrupted Heavy Gunner Specter"
+    "value": "5X Corrupted Heavy Gunner Specter"
```
Comment (TobiTenno) [inline data/languages.json]: lowercase the X, plz.

File: data/languages.json [hunk 3]
```diff
@@ -5296,7 +5296,7 @@
     "value": "Grustrag Three Beacon"
   },
   "/lotus/storeitems/types/restoratives/consumable/barofireworkscrate": {
-    "value": "Ki'Teer Fireworks"
+    "value": "10X Ki'Teer Fireworks"
```
Comment (TobiTenno) [inline data/languages.json]: lowercase X, please

File: data/languages.json (hunk 1 above)
Comment (cagatayldzz) [reply data/languages.json]: I'm checking

Comment (cagatayldzz) [reply data/languages.json]: I couldn't see a problem

Comment (TobiTenno) [reply data/languages.json]: The screenshot says ankle

The json says Leg

**Pull request merged**